packages/contentful--app-scripts/lib/upload/build-upload-settings.js

```diff
@@ -22,6 +23,21 @@ async function buildAppUploadSettings(options) {
       default: '',
     });
   }
+  if (!skipActivation) {
+    prompts.push({
+      type: 'confirm',
+      name: 'skipActivation',
+      message: `Skip bundle activation after upload?`,
```

Comment:
🤔  To me it feels like this question would be easier to understand in reverse:

> Do you want to activate the bundle after upload?

Reply:
Does it work now? We set `skipActivation` to true if we want to activate the bundle. I feel like we need to map from the user input to `skipActivation`

Reply:
Oh, yes, good catch. Fixed it now.